FakeXrmEasy.Tests.Shared/FakeContextTests/QualifyLeadTests/QualifyLeadTests.cs

```diff
@@ -139,5 +139,36 @@ public void Check_if_Account_was_associated_with_Opportunity()
 
             Assert.NotNull(opportunity.CustomerId);
         }
+
+        [Fact]
+        public void Status_of_qualified_Lead_should_be_qualified()
+        {
+            var context = new XrmFakedContext();
+            context.ProxyTypesAssembly = Assembly.GetExecutingAssembly();
```

Comment:
There's also no need for this line if early bound entities are passed (Lead) to the Initialize method, it'll discover the assembly for us via reflection :)

```suggestion

```